docs/release-notes/2.17.0.md

```diff
@@ -0,0 +1,74 @@
+# Percona Monitoring and Management 2.17.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">May 10, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+- Custom Certificates support for MySQL monitoring, including remote monitoring, will let you monitor MySQL instances with such security restrictions. This functionality might help monitor Google Cloud SQL for MySQL instances easier.
+
+- External monitoring cleanups on UI to make it easier to use. You can fill all parameters or provide the URL of your endpoint, and PMM will detect all needed parts.
+
+- pg_stat_monitor 0.9.0 support. This change will give you compatibility with the latest version. More features coming later.
+
+- Single-line install of PMM Server on supported Linux distros (this feature is in Technical Preview).
+
+- DBaaS Changes: (this feature is in Technical Preview)
+    - Make it easier to preview DBaaS functionality, so users can quickly turn it ON on the Advanced settings page. ([Read more](../how-to/configure.md#advanced-settings))
```

Comment:
```suggestion
    - Make it easier to tinker with DBaaS functionality; users can quickly turn it ON/OFF on the Advanced settings page. ([Read more](../how-to/configure.md#advanced-settings))
```

Reply:
'tinker with' is what you do to a pre-war Austin. Maybe 'adjust'

Reply:
'experience'